src/views/Dashboard/Customer/Home/Emr/Create.vue

```diff
@@ -494,22 +502,44 @@ export default {
       }
     },
 
+    handleShowPassword() {
+      this.showPassword = !this.showPassword
+    },
+
+    async fetchWalletBalance() {
+      try {
+        return await queryBalance(this.api, this.wallet.address)
+      } catch (err) {
+        console.error(err.message)
+      }
+    },
+
     async finalSubmit() {
       this.isLoading = true
+
       try {
+        const balance = await this.fetchWalletBalance()
+
         await this.wallet.decodePkcs8(this.password)
 
+        if (balance < 10) {
```

Comment:
this error could be handle by handling error from RPC substrate :
```
 RPC-CORE: submitExtrinsic(extrinsic: Extrinsic): Hash:: 1010: Invalid Transaction: Inability to pay some fees , e.g. account balance too low
/Users/lydia/Documents/blocksphere/debio/debio-dummy-data-injector/node_modules/@polkadot/api/node_modules/@polkadot/rpc-provider/coder/index.cjs:84
```

Reply:
Sure, will update asap 👍🏼